File: edudoc/src/Model/DTOs/StudentParentalConsentDTO.cs
using System.Data.SqlClient;
using System.Data.SqlClient;

using System;

namespace Model.DTOs
{
    public class StudentWithParentalConsentDTO
    {
        public int Id { get; set; }
        public string LastName { get; set; }
        public string FirstName { get; set; }
        public string StudentCode { get; set; }
        public School School { get; set; }
        public string Grade { get; set; }
        public StudentParentalConsentType Consent { get; set; }
        public DateTime DateOfBirth { get; set; }
        public int TotalBillableClaims { get; set; }
        public DateTime? EffectiveDate { get; set; }
    }
}

File: edudoc/src/app/model/interfaces/custom/student-parental-consent.dto.ts
import { ISchool } from '../school';
import { IStudentParentalConsentType } from '../student-parental-consent-type';

export interface IStudentParentalConsentDTO {
    Id: number;
    LastName: string;
    FirstName: string;
    DateOfBirth: Date;
    StudentCode: string;
    School: ISchool;
    Grade: string;
    Consent: IStudentParentalConsentType;
    TotalBillableClaims: number;
    EffectiveDate: Date;
}
